### PR TITLE
Code clean-up

### PR DESCRIPTION
**Pull request by Kami-no**
Clean code with 2 spaces ident and no trailing spaces

### CODE REVIEW

**Comment by rimas-kudelis** [comment]
I would prefer PSR-compliant code (it uses 4 spaces per indentation step, if I remember correctly), but do you think it's really worth it? This codebase is anything but clean either way...

**Comment by Kami-no** [comment]
PSR - no problem. I prefer it too but in your code 2 spaces are used more often. I'll change it. I'm going to clean all code anyway. Clean code helps to read the code.

I'm interested in vexim at CentOS for the project in my company. I'd like to make some blacklists and whitelists.

I just offer my changes to upstream.

**Comment by Kami-no** [comment]
Nice clean code now. By the way, I've found several times code after die. IDE noticed that it will never be executed. Next set of updates getting ready but they're not for this pull.

File: vexim/adminaliaschange.php [hunk 1]
```diff
@@ -1,144 +1,145 @@
...
-		<?php 		
-			# end of the block editing an alias within the domain
-		}  
-		?>	
-    </div>
-  </body>
+                    </td>
+                </tr>
+                <tr
+                    >
```
Comment (rimas-kudelis) [inline vexim/adminaliaschange.php]: You seem to have missed this.

File: vexim/adminfailchange.php [hunk 2]
```diff
@@ -1,66 +1,67 @@
...
+<body onLoad="document.failchange.localpart.focus()">
+<?php include dirname(__FILE__) . '/config/header.php'; ?>
+<div id="menu">
+    <a href="adminfail.php"><?php echo _('Manage Fails'); ?></a><br>
+    <a href="adminfailadd.php"><?php echo _('Add Fail'); ?></a><br>
+    <a href="admin.php"><?php echo _('Main Menu'); ?></a><br>
+    <br><a href="logout.php"><?php echo _('Logout'); ?></a><br>
+</div>
+<div
+="Forms">
```
Comment (rimas-kudelis) [inline vexim/adminfailchange.php]: Obviously an accident.

**Comment by rimas-kudelis** [comment]
But seriously, I really doubt that fixing indentation makes this code any better. I think I see it as being in maintenance phase where only bugfixes are to be applied when necessary. If you want to remove closing PHP tags or fix bugs in the code, that's OK, but I don't think that changing indentation of 95% of the code makes sense at its current state.
Oops, I'm repeating myself... :(

**Comment by Kami-no** [comment]
No problem to me if you would not accept the Pull Request.

**Pull request closed without merging**